app/src/main/java/com/vero/tasky/core/presentation/navigation/RootNavigation.kt

```diff
@@ -80,7 +83,7 @@ fun RootNavigation(
             arguments = agendaItemArgumentTypes,
             deepLinks = listOf(
                 navDeepLink {
-                    uriPattern = "$EVENT_DEEP_LINK{$ITEM_ID}"
+                    uriPattern = "$DEEP_LINK_HOST?$EVENT_ITEM_ID={$EVENT_ITEM_ID}"
```

Comment:
I'd make a constant or similar for the deeplink (or a function to generate it)

Since you will use it in couple of places (here + notification builder)
So it's prone to error if typed manually